Building dependency in Fusesoc step fails on Ubuntu 20.04

<!--
Thank you for your interest in Ibex. To help us understand your question as quickly as possible we have prepared a little template below.

No text in blocks starting with < !-- will be shown in the final bug report.

You can also copy/paste images into the question, and you can attach files to it by simply dragging them into the issue window.
-->

<!--
Please ask your question after this block of text, adding as much detail as possible.
If your question is related to a specification (e.g. the RISC-V Specification), a part of our source code, or our documentation we appreciate a link or a quote from the relevant text.
-->

Hii, I am trying to run the ibex core on my Ubuntu 20.04 system as suggested in https://github.com/lowRISC/ibex/tree/master/examples/simple_system. 

The dependencies are correctly installed, but the fusesoc building simulation step fails inspite of that:
fusesoc --cores-root=. run --target=sim --setup --build lowrisc:ibex:ibex_simple_system --RV32E=0 --RV32M=ibex_pkg::RV32MFast

The error is as follows:

INFO: Running pre_build script check_tool_requirements
INFO: Tool verilator present: Sufficiently recent version (found 4.213; needed 4.028)
INFO: Tool edalize present: Sufficiently recent version (found 0.2.2; needed 0.2.0)
INFO: Building simulation model
verilator -f lowrisc_ibex_ibex_simple_system_0.vc --trace --trace-fst --trace-structs --trace-params --trace-max-array 1024 -CFLAGS "-std=c++11 -Wall -DVM_TRACE_FMT_FST -DTOPLEVEL_NAME=ibex_simple_system -g" -LDFLAGS "-pthread -lutil -lelf" -Wall -Wwarn-IMPERFECTSCH --unroll-count 72
%Warning-UNUSED: ../src/lowrisc_ibex_ibex_top_tracing_0.1/rtl/ibex_top_tracing.sv:27:26: Parameter is not used: 'RndCnstLfsrSeed'
                                                                                       : ... In instance ibex_simple_system.u_top
   27 |   parameter lfsr_seed_t  RndCnstLfsrSeed  = RndCnstLfsrSeedDefault,
      |                          ^~~~~~~~~~~~~~~
                 ... For warning description see https://verilator.org/warn/UNUSED?v=4.213
                 ... Use "/* verilator lint_off UNUSED */" and lint_on around source to disable this message.
%Warning-UNUSED: ../src/lowrisc_ibex_ibex_top_tracing_0.1/rtl/ibex_top_tracing.sv:28:26: Parameter is not used: 'RndCnstLfsrPerm'
                                                                                       : ... In instance ibex_simple_system.u_top
   28 |   parameter lfsr_perm_t  RndCnstLfsrPerm  = RndCnstLfsrPermDefault,
      |                          ^~~~~~~~~~~~~~~
%Error: Exiting due to 2 warning(s)
Makefile:16: recipe for target 'Vibex_simple_system.mk' failed
make: *** [Vibex_simple_system.mk] Error 1
ERROR: Failed to build lowrisc:ibex:ibex_simple_system:0 : 'make' exited with an error code


I am new to ibex, can anyone be kind to help me with the resolution of the error. 
Thanks in advance.





## My Environment

<!--
Let us know how you're using Ibex. This helps us to better understand your problem and find a solution more quickly.
-->

**EDA tool and version:**
<!--
If you are using an EDA tool to work with Ibex let us know which tool and which version you're using.
For example, "Xilinx Vivado 2020.1" or "Synopsys VCS 2020.03-SP1"
-->

**Operating system:**
<!--
Please let us know the operating system/Linux distribution you're using.
For example "Ubuntu Linux 18.04" or "CentOS 7.3".
-->

**Version of the Ibex source code:**
<!--
Which version of the Ibex source code are you using?
You can call `git rev-parse HEAD` in your Ibex source directory to get this information.
Please also let us know if you have made changes to the code.
-->

--- a/rtl/ibex_top_tracing.sv
+++ b/rtl/ibex_top_tracing.sv
@@ -129,6 +129,8 @@ module ibex_top_tracing import ibex_pkg::*; #(
     .DbgHwBreakNum    ( DbgHwBreakNum    ),
     .WritebackStage   ( WritebackStage   ),
     .SecureIbex       ( SecureIbex       ),
+    .RndCnstLfsrSeed  ( RndCnstLfsrSeed  ),
+    .RndCnstLfsrPerm  ( RndCnstLfsrPerm  ),
     .DmHaltAddr       ( DmHaltAddr       ),
     .DmExceptionAddr  ( DmExceptionAddr  )
   ) u_ibex_top (

